src/js/components/award/financialAssistance/FinancialAssistanceDetails.jsx

```diff
@@ -61,24 +61,13 @@ export default class FinancialAssistanceDetails extends React.Component {
 
         // Location
         let popPlace = "Not Available";
-        let popZip = null;
-        if (award.pop_zip) {
-            popZip = award.pop_zip;
+        if (award.pop_country_code === 'USA') {
+            popPlace = `${award.pop_city}, ${award.pop_state_province} ${award.pop_zip}`;
+            popPlace +=
+            `\nCongressional District: ${award.pop_state_code}-${award.pop_congressional_district}`;
```

Comment:
Same, what if there is no state code or congressional district (see `award/603`)?